### PR TITLE
lookaway 1.4.1 (new cask)

### PR DESCRIPTION
**Pull request by kushsolitary**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by kushsolitary** [comment]
Hello, we use github only to host the binaries but the audit is failing because the repository is not notable enough. The app has its own website: https://lookaway.app and is getting quite popular.

**Comment by kushsolitary** [comment]
Hello, we use github only to host the binaries but the audit is failing because the repository is not notable enough. The app has its [own website](https://lookaway.app/) and is getting quite popular.

**Comment by krehel** [comment]
Congrats on opening you first PR in Homebrew.

Seems like we aren't allowed to edit this PR.  Any particular reason why?  Helpful if you allow maintainer edits, if needed.

File: Casks/l/lookaway.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  url "https://github.com/mysticalbits/lookaway-releases/releases/download/#{version}/LookAway.dmg",
+      verified: "github.com/mysticalbits/lookaway-releases/"
+  name "LookAway"
+  desc "Reduce dry eyes and digital strain"
+  homepage "https://lookaway.app/"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```
Comment (krehel) [inline Casks/l/lookaway.rb:16]: Not needed if you don't use pre-releases, and preferred not to have it.

```suggestion
```

File: Casks/l/lookaway.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  depends_on macos: ">= :monterey"
+
+  app "LookAway.app"
+
+  uninstall quit: "com.mysticalbits.lookaway"
+
+  zap trash: "~/Library/Application Support/LookAway"
```
Comment (krehel) [inline Casks/l/lookaway.rb:24]: Found other files

```suggestion
  zap trash: [
    "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/com.mysticalbits.lookaway.sfl*",
    "~/Library/Application Support/LookAway",
    "~/Library/Caches/SentryCrash/LookAway",
    "~/Library/Preferences/com.mysticalbits.lookaway.plist",
  ]
```

File: Casks/l/lookaway.rb (hunk 1 above)
Comment (kushsolitary) [reply Casks/l/lookaway.rb]: fixed!

**Comment by kushsolitary** [comment]
> Congrats on opening your first PR in Homebrew!
> 
> Seems like we aren't allowed to edit this PR. Any particular reason why? Helpful if you allow maintainer edits, if needed.

hmm.. I am actually not sure how/where to enable that. Could you please point me to the right direction?

File: Casks/l/lookaway.rb (hunk 2 above)
Comment (kushsolitary) [reply Casks/l/lookaway.rb]: thank you, added these as well.

File: Casks/l/lookaway.rb [hunk 3]
```diff
@@ -10,6 +10,7 @@
   desc "Reduce dry eyes and digital strain"
   homepage "https://lookaway.app/"
 
+  auto_updates true
```
Comment (krehel) [inline Casks/l/lookaway.rb:13]: Thanks for adding `auto_updates`, I was going to ask about this.

File: Casks/l/lookaway.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+cask "lookaway" do
+  version "1.4.1"
+  sha256 "0f211bad3302453f28bbde874ede70e5c048d1ff09e8d53eadbb977b40a9e0b3"
+
+  url "https://github.com/mysticalbits/lookaway-releases/releases/download/#{version}/LookAway.dmg",
+      verified: "github.com/mysticalbits/lookaway-releases/"
+  name "LookAway"
+  desc "Reduce dry eyes and digital strain"
```
Comment (krehel) [inline Casks/l/lookaway.rb:10]: Maybe something like the first option?  I was working on the second option below but doesn't really tell what the application is for.  Thoughts? @kushsolitary?
```suggestion
  desc "Break time reminder app"
```
```suggestion
  desc "Reduce digital eye strain"
```

Comment (kushsolitary) [reply Casks/l/lookaway.rb:10]: The first options sounds good to me @krehel

**Comment by bevanjkay** [comment]
> Edit: I don't have the option "Allow edits from maintainers" which is supposed to be here

For future reference, this is because the PR is opened from an org fork.

File: Casks/l/lookaway.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
```
Comment (bevanjkay) [inline Casks/l/lookaway.rb:2]:
```suggestion
```

This should be removed.

**Review by bevanjkay — CHANGES_REQUESTED** [review]

**Review by bevanjkay — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @kushsolitary.

**Pull request merged**